intercom_flutter_platform_interface/analysis_options.yml

```diff
@@ -0,0 +1,4 @@
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
```

Comment:
Why we need to add this analyzer?

Reply:
It was there for the main plugin, so I added it also for the web and platform interface